_posts/en/newsletters/2021-01-27-newsletter.md

```diff
@@ -0,0 +1,89 @@
+---
+title: 'Bitcoin Optech Newsletter #133'
+permalink: /en/newsletters/2021/01/27/
+name: 2021-01-27-newsletter
+slug: 2021-01-27-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter announces a meeting to discuss taproot activation
+mechanisms and includes our regular sections with top questions and
+answers from the Bitcoin StackExchange, a list of releases and release
+candidates, and descriptions of notable changes to popular Bitcoin
+infrastructure software.
+
+## News
+
+- **Scheduled meeting to discuss taproot activation:** Michael Folkson
+  [announced][folkson announce] that a meeting will be held at <time
+  datetime="2021-02-02 19:00-0000">19:00 UTC on February 2nd</time> in
+  the [##taproot-activation][] IRC channel on irc.freenode.net to
+  discuss some desired revisions to [BIP8][].  It has not yet been
+  decided that BIP8 will actually be used for the activation, so
+  alternative proposals may also be discussed during the meeting or at a
+  subsequent meeting.  Please see Folkson's email for background
+  information about taproot activation mechanisms and a proposed agenda
+  for the meeting.
+
+## Selected Q&A from Bitcoin StackExchange
+
+*[Bitcoin StackExchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+FIXME:bitschmidty
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+<!-- FIXME:harding to update tuesday -->
+
+- [C-Lightning 0.9.3][c-lightning 0.9.3] is the project's newest minor release.
+  It includes a several improvements to the user interface and plugin
+  capabilities, plus experimental support for the proposed onion
+  messages protocol (see [Newsletter #92][news92 cl3600]) and offers protocol (see
```

Comment:
(feel free to ignore, but...) are "onion messages protocol" and "offers protocol" proper nouns that should be capitalized or set off with italics or quotes? if not, maybe s/offers protocol/the offers protocol/ (which they have written as well) to disambiguate a word that can be a noun, a verb, and here is a name or proper noun of sorts